File: C/1171.c
#include <stdio.h>

int main() {

    int num[2001] = {0};
    int n, x;
    scanf("%d",&n);

    for(int i = 0 ; i < n ; i++){
        scanf("%d", &x);
        num[x]++;
        }

    for(int i = 0; i < 2001; i++){
        if(num[i] > 0){
            printf("%d aparece %d vez(es)\n",i,num[i]);
            }
        }

    return 0;
}

File: Python/1171.py
lista = {}
for i in range(int(input())):
  entrada = int(input())
  if entrada in lista: lista[entrada] += 1 
  else: lista[entrada] = 1

a = list(lista.keys())
a.sort()

for chave in a:
    print(f"{chave} aparece {lista[chave]} vez(es)")
